docs/setup/custom-config.mdx

```diff
@@ -44,40 +39,99 @@ python -m mindsdb --no_studio
 
 ## Starting MindsDB with Extended Configuration
 
-First, you should prepare a `config.json` file based on the following template; remember to substitute the values for your custom configuration.
+Start MindsDB locally with your custom configuration by providing a path to the `config.json` file that stores custom config parameters listed in this section.
+
+```bash
+python -m mindsdb --config=/path-to-the-extended-config-file/config.json
+```
+
+Below are all of the custom configuration parameters that should be set according to your requirements and saved into the `config.json` file.
 
 ```bash
 {
     "permanent_storage": {
         "location": "local"
+        "bucket": "s3_bucket_name" # optional
     },
+```
+
+The `permanent_storage` parameter defines where MindsDB stores essential user files, such as uploaded files, models, and tab content, providing an active storage mechanism. MindsDB checks the `permanent_storage` location to access the latest version of a file and updates it as needed.
+
+The `location` specifies the storage type.
+
+* `absent` (default): Disables permanent storage and is not recommended to use.
```

Comment:
It is **recommended** to use if use mindsdb locally. Any other option may be required in rare cases only.

Reply:
Updated it to:
```
* `absent` (default): Disables permanent storage and is recommended to use when MindsDB is running locally.
```